src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -1230,22 +1212,6 @@ private Collection<FunctionMetadata> LoadProxyRoutes(string proxiesJson)
             error = null;
             functionMetadata = ParseFunctionMetadata(functionName, functionConfig, scriptDirectory, settingsManager);
 
-            if (functionMetadata.IsExcluded)
-            {
-                string message = $"Function '{functionName}' is marked as excluded";
-                traceWriter.Info(message);
-                logger?.LogInformation(message);
-                functionMetadata = null;
-                return true;
-            }
-
-            if (functionMetadata.IsDisabled)
-            {
-                string message = $"Function '{functionName}' is disabled";
```

Comment:
I would be good to carry this log forward somehow, at the point in time during startup where we know this. It's useful in the logs to see what functions are disabled

Reply:
We have logic to log disabled function in SDK:

https://github.com/Azure/azure-webjobs-sdk/blob/fa20618c420b1fc7173afdaa4e8678c5aab1a5ae/src/Microsoft.Azure.WebJobs.Host/Listeners/HostListenerFactory.cs#L55

+ send PR to log HttpTriggers as well:
https://github.com/Azure/azure-webjobs-sdk/pull/1428

Reply:
SDK's PR checked in